src/styles/globals.css

```diff
@@ -123,3 +123,20 @@ body {
 .ais-SearchBox input {
   @apply h-full w-full rounded-xl border-2 border-primary/30 bg-transparent p-4 text-3xl outline-none placeholder:text-base-content focus:ring-2 focus:ring-primary;
 }
+
```

Comment:
@ujjawaltyagii We are using tailwindCSS so you are not required to make changes in global CSS. use utility based classes for navbar and remove the CSS from global.css.
Commit those changes.

Reply:
done !!